prog/vnet/update_load_balancer_node.rb

```diff
@@ -12,7 +12,7 @@ def before_run
   end
 
   label def update_load_balancer
-    vm.load_balancer_vm_ports.select { |lvp| lvp.state == "detaching" }.each do |load_balancer_vm_port|
+    vm.load_balancer_vm_ports.select { |lvp| lvp.state == "detaching" }.sort_by(&:stack).each do |load_balancer_vm_port|
```

Comment:
`vm.load_balancer_vm_ports_dataset.where(state: "detaching").order(:stack).all do |load_balancer_vm_port|`

This could theoretically remove multiple ports at once for a dual stack load balancer, so I'm guessing this loop could result in duplicate queries.